pkg/graveler/graveler.go

```diff
@@ -912,6 +912,11 @@ type CommittedManager interface {
 
 	// GetRangeIDByKey returns the RangeID that contains the given key.
 	GetRangeIDByKey(ctx context.Context, ns StorageNamespace, id MetaRangeID, key Key) (RangeID, error)
+
+	// Import applies changes from 'source' to 'destination' and returns the ID of the new metarange.
+	// This is similar to the Merge operation, but it will override existing prefixes in the destination
```

Comment:
I wouldn't mention it being similar to the merge operation, since merge operates on commits and here we operate on metaranges. Metaranges don't have parents, so it's not well-defined to talk about a merge here.